src/Controller/ListController.php

```diff
@@ -96,4 +97,19 @@ public function getMembersAction(Request $request, SubscriberList $list): View
 
         return View::create()->setData($list->getSubscribers());
     }
+
+    /**
+     * Gets the total number of subscribers of a list.
+     *
+     * @param Request $request
+     * @param SubscriberList $list
+     *
+     * @return View
+     */
+    public function getCountAction(Request $request, SubscriberList $list): View
+    {
+        $this->requireAuthentication($request);
+
+        return View::create()->setData(count($list->getSubscribers()));
```

Comment:
Will this actually retrieve all subscribers? Or will this only get the number of subscribers? For performance reasons, I'd prefer the latter.

Reply:
@xh3n1 Have you researched this? (I'd like to avoid all subscriber data getting loaded just for getting the number of subscribers. For this, changes to the core might be required.)

Reply:
@oliverklee yes, I think it is good for now. Changes for performance improvements can be done in another PR after further research is done, also related to #117

Reply:
@xh3n1 And what was the result of your research? Do all models get loaded into memory, or are they lazy-loaded?

Reply:
@oliverklee I'm not a doctrine expert but would the use of "Extra-Lazy Associations" improve the performance? https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/tutorials/extra-lazy-associations.html
Here's the Subscribers collection used by the function:
https://github.com/phpList/core/blob/master/src/Domain/Model/Messaging/SubscriberList.php#L112-L122

Reply:
@xh3n1 To clarify: not all subscriber data is retrieved in order to execute the action, correct? Only the count is sought and returned, so that unnecessary DB reads are avoided?

Reply:
@samtuke Only the count is returned but:
to my understanding of the documentation:  'subscribers' collection is marked as **lazy** by default, which means the whole collection for this association is loaded the first time it's accessed. If you mark an association as **extra lazy** the following method on collections can be called without triggering a full load of the collection, it will just issue a COUNT query to the database.

**_Collection#count()_**

is that correct @oliverklee ?

Reply:
I don't know. I propose we check this by logging the MySQL queries for this. @xh3n1 Would you be willing to do this?

Reply:
Hi @oliverklee 
I tested it further by logging the MySQL queries. 
If I add the "fetch="EXTRA_LAZY", the following query will be executed:
	
```
SELECT COUNT(*) FROM phplist_listuser t WHERE t.listid = 10
```
which doesn't load any extra data in the memory, thus not causing a PHP time limit exception even for a large number of subscribers such as over 54000. 
So, I will create a PR for to add EXTRA_LAZY in core repo.

FYI @samtuke

Reply:
PR :https://github.com/phpList/core/pull/306

Reply:
Reviewed and merged.